cli/broadlink_cli

```diff
@@ -126,7 +126,7 @@ if args.send:
     data = durations_to_broadlink(parse_durations(' '.join(args.data))) \
         if args.durations else bytearray.fromhex(''.join(args.data))
     dev.send_data(data)
-if args.learn or args.learnfile:
+if (args.learn or args.learnfile) and not args.rfscanlearn:
```

Comment:
There is no reason to add this conditional because these arguments are exclusive. The user will never use both on the same call.

Reply:
Doesn't it force a "learnfile" parameter here? https://github.com/mjg59/python-broadlink/blob/a731c9c5a5d03b6894a882c4a8eaf5e0b333296a/cli/broadlink_cli#L236

Reply:
Ah yes, I understand now. Looks good to me. A suggestion:
```python
if args.learn or (args.learnfile and not args.rfscanlearn):
```